specs/swagger.yaml

```diff
@@ -104,7 +104,32 @@ paths:
               schema:
                 $ref: "#/components/schemas/StateReturnWithTime" 
         "400":
-          description: Session not active, or model not initialized 
+          description: Session not active, or model not initialized
+  /v1/session/{id}/output: 
+    parameters:
+      - name: id
+        description: id for the active session (generated when session began)
+        in: path 
+        required: true
+        schema:
+          type: integer
+      - name: return_format
+        description: Format for the returned state
+        in: query
+        required: false
+        schema:
+          $ref: "#/components/schemas/ReturnFormat"
+    get:
+      description: Get the most recent estimate of the system output for the model used in this session.
+      responses:
+        "200":
+          description: Okay response
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/StateReturnWithTime" 
```

Comment:
Should there be an "OutputReturnWithTime", or does this follow the same schema? I'm not sure what to name it if the one schema is for both State and Output... "VariableReturnWithTime"?

Reply:
Perhaps "state return" is misleading. The same structure is used for any prediction. 
 
 I've renamed the StateReturn structure to be "PredictionPoint" so that it's not misleading. What do you think?